source/StepBro.ConsoleSidekick.WinForms/MainForm.cs

```diff
@@ -1097,6 +1110,7 @@ private void SetToolbarVisibility(string name, bool visible)
 
         private void UpdateToolbarVisibility()
         {
+            int missingHeight = toolStripMain.Bounds.Bottom - this.ClientRectangle.Height;
```

Comment:
Move this to after the visibility check.

Reply:
Where do you mean?

If we put it inside the if-statement, then it is not visible when we get to the end of the function where we use it if there are no custom toolbars.

Do you just mean after the next line or?